packages/react-codemirror/src/lang-cypher/autocomplete.ts

```diff
@@ -77,13 +77,13 @@ export const cypherAutocomplete: (config: CypherConfig) => CompletionSource =
     }
 
     const options = autocomplete(
-      textUntilCursor,
+      documentText,
       config.schema ?? {},
-      undefined,
+      context.pos,
```

Comment:
This was the reason why it was working in codemirror but not in VSCode, we were reparsing the query. This should avoid reparses